app/assets/stylesheets/bootstrap_and_overrides.css.less

```diff
@@ -97,3 +229,14 @@ table {
   }
 }
 
+
+/* This is necessary to correctly display the Glyphicons icons
+-------------------------------------------------- */
+
+@font-face {
+  font-family: 'Glyphicons Halflings';
+  src: url('/assets/glyphicons-halflings-regular.eot');
```

Comment:
This will probably not work on production. On production each asset is compiled into a file with the hash of the content included in the filename. And hence the filename will be different that in development.